dashboard/app/models/pd/regional_partner_contact.rb

```diff
@@ -17,6 +17,8 @@
 class Pd::RegionalPartnerContact < ActiveRecord::Base
   include Pd::Form
 
+  UNMATCHED_FORM_EMAIL = 'partner@code.org'
```

Comment:
I'm sure you're doing this with tests now, but last time we tried this sending to partner@ didn't work. This alias doesn't have a gmail account, it gets sent to zendesk. But for whatever reason no email was ever making it to zendesk. I think we have to send it to an individual, and I think we'd change it from Tawny to Anthonette.

Reply:
Good catch.  I don't have tests for this, because my tests don't actually send email through our GSuite system.  I just sent an email to `partner@code.org` to see what happens. Should we check with the other stakeholders about what they'd like to do here?

Reply:
Looks like it went to Zendesk and auto-assigned to Anthonette.

Reply:
Yeah, so sending an email directly to `partner@code.org` works. But for some reason when we sent it through our amazon automated system, it didn't work.

+@poorvasingal I think this email will need to be addressed directly to Anthonette. She is the one managing the partner@ alias anyway, so it would still have the same outcome.

Reply:
I'm going to test that out again - sending an email via this system (on production-console) to `partner@code.org` and see what happens.

Reply:
Hmm... indeed, it seems this mail did not reach Zendesk.  Here's the output I get when sending this mail from dashboard-console.
```
[production] dashboard > Pd::RegionalPartnerContactMailer.unmatched(form.merge(notes: 'This is a test email from Brad to see if mail sent from our production mailing system can go to the partner@ alias'), 'partner@code.org')
Pd::RegionalPartnerContactMailer#unmatched: processed outbound mail in 346.3ms
=> #<Mail::Message:637280420, Multipart: false, Headers: <From: Tanya Parker <tanya_parker@code.org>>, <To: partner@code.org>, <Subject: A district administrator wants to connect with Code.org>, <Mime-Version: 1.0>, <Content-Type: text/html>>
```
Nothing obviously wrong there, but it doesn't show up in Zendesk.

Looking at the configuration in Zendesk, there is a warning:
![image](https://user-images.githubusercontent.com/1615761/46381996-95a9ee00-c65d-11e8-84cd-79eb711838c5.png)

Reply:
Looks like [using an email/distribution list as a support address is not supported.](https://support.zendesk.com/hc/en-us/articles/115004306927-Is-using-an-email-alias-distribution-list-or-Google-group-as-a-support-address-supported-)